curriculum/js0/5.js

```diff
@@ -7,9 +7,17 @@
  */
 
 const solution = (a, b, c) => {
-  return 0;
-};
 
-module.exports = {
-  solution,
+  if (a>b && a>c) {
+    return a;
+  }
+  if (b>a && b>c) {
+    return b;
+  }
+  if (c>a && c>b) {
+    return c;
+  }
+}
+module.exports ={
```

Comment:
Another way to simplify your code:

const solution = (a, b, c) => {
  if (a>b && a>c) {
    return a;
  }
  if (b>a && b>c) {
    return b;
  }
    return c;
}